src/api/merchant/login.js

```diff
@@ -0,0 +1,39 @@
+import axios from "axios";
```

Comment:
this whole file is not needed. all the login/register of both customer and merchant will be inside user folder and respective login or register routes